mods/ura/rules/civilian.yaml

```diff
@@ -142,7 +142,12 @@ V01:
 		Type: Rectangle
 			TopLeft: -768, -597
 			BottomRight: 896, 683
-
+V38:
+	Inherits: V01
+	SpawnActorOnDeath:
+		Actor: moneycrate
+	RenderSprites: 
+            Image: V01
```

Comment:
needs indentation fix

Reply:
might also wanna drop in something like 'Tooltip: Church Moneycrate' to distinguish it in the map editor